docs/robot.rst

```diff
@@ -1,6 +1,6 @@
-===============
-Robot Framework
-===============
+========================
+Automate Robot Framework
```

Comment:
This title sounds a bit weird to me. Robot Framework is a tool that we use to automate other things, not a thing that we automate.

Reply:
@davisagli @davidmreed @boakley - Do we like "Integrate Robot Framework", or just "Robot Framework"? I'm open to other suggestions as well.

Reply:
How about "Automate Using Robot Framework"?

also I just noticed that "Automating Data Operations" should probably be "Automate Data Operations" to match the form of the other verbs in section titles

Reply:
How about "Automated browser testing with Robot Framework"?

Reply:
Robot Framework can also be used to automate non-browser actions, but I'm okay with that if it's a good description of what's covered in that section.

Reply:
> How about "Automate Using Robot Framework"?
> 
> also I just noticed that "Automating Data Operations" should probably be "Automate Data Operations" to match the form of the other verbs in section titles

This section is already named "Automate Data Operations".

Reply:
> data.rst already has the title "Automate Data Operations".

Sorry, I think I was looking at the wrong branch on readthedocs